firefox-ios/Client/Frontend/Library/Bookmarks/BookmarksTelemetry.swift

```diff
@@ -0,0 +1,32 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Glean
+
+struct BookmarksTelemetry {
+    private let gleanWrapper: GleanWrapper
+
+    enum EventLabel: String {
```

Comment:
I'm not sure which is the correct way of calling `EventLabel` or `EventValue` it's called indistinctly between the TelemetryWrapper and GleanMetrics generated file

Reply:
I think `EventLabel` makes sense